fahrenheit-404.py

```diff
@@ -0,0 +1,45 @@
+while True:
+    print("Hello, Please Select The Temperature You Want To Convert From\nOption 1: Fahrenheit\nOption 2: Celsius\nOption 3 Kelvin")
+    from_temp = input("Please select Option 1, Option 2, or Option 3\n").lower()
+    print("Please, Select The Temperature You Would Like to Convert To\nOption 1: Fahrenheit\nOption 2: Celsius\nOption 3 Kelvin")
+    to_temp = input("Please select Option 1, Option 2, or Option 3\n").lower()
+    if from_temp == to_temp:
```

Comment:
Should this be a single equals sign?

```suggestion
    if from_temp = to_temp:
```